plasmapy/mathematics/mathematics.py

```diff
@@ -155,7 +155,7 @@ def plasma_dispersion_func_deriv(zeta):
 
     return Zprime
 
-
+#%%
```

Comment:
Wut? :D

Reply:
Oh, damn it I thought I removed all my testing code.
`#%%` is a way of creating cells which can be individually run in Spyder.

I've now removed it in my local code.